src/OrchardCore.Modules/OrchardCore.Deployment/ViewModels/RecipeFileDeploymentStepViewModel.cs

```diff
@@ -0,0 +1,8 @@
+using OrchardCore.Deployment.Steps;
+
+namespace OrchardCore.Deployment.ViewModels
+{
+    public class RecipeFileDeploymentStepViewModel: RecipeFileDeploymentStep
+    {
```

Comment:
Not sure about inheriting like this. While it will work view models, and models are intended to be separated for a reason (binding).
I think it was fine as it is

Reply:
My intend to avoid the redundancy